game/src/core/systems/LevelSystem.java

```diff
@@ -173,7 +173,7 @@ private void drawLevel() {
                 if (t.levelElement() != LevelElement.SKIP) {
                     String texturePath = t.texturePath();
                     if (!mapping.containsKey(texturePath)) {
-                        mapping.put(texturePath, new PainterConfig(texturePath));
+                        mapping.put(texturePath, new PainterConfig(texturePath, 0.5f, 0.25f));
```

Comment:
magic number für das offset.
Kriegen wir das in zwei Konstanten ausgelagert? Sonst raft das niemand in Zukunft mehr